src/config/dynamic-theme-fixes.config

```diff
@@ -9752,6 +9752,8 @@ span[class^="ts-"]
 img[src$="Rename_icon_Cyr.svg.png"]
 img[alt="Aksara Sunda.png"]
 img[alt="Aksara kawi name.png"]
+img[src*="/static/images/mobile/copyright/wikipedia-wordmark"]
```

Comment:
It seems like `.header .branding-box > a > span > img` on line 9743 have the same effect.